cisco-nexus-3048TP/workflows/deploy-startup-config-boot-images-graph.json

```diff
@@ -0,0 +1,56 @@
+{
+    "friendlyName": "Deploy Cisco Nexus 3000 switch",
+    "injectableName": "Graph.Switch.CiscoNexus3000.Deploy",
+    "options": {
+        "deploy-config-and-images": {
+            "startupConfig": null,
+            "kickstartImage": null,
+            "bootImage": null
+        }
+    },
+    "tasks": [
+        {
+            "label": "deploy-config-and-images",
+            "taskDefinition": {
+                "friendlyName": "Deploy switch config and images",
+                "injectableName": "Task.Inline.Switch.Deploy.Cisco",
+                "implementsTask": "Task.Base.Linux.Commands",
+                "options": {
+                    "startupConfig": null,
+                    "startupConfigUri": "{{ api.base }}/templates/{{ options.startupConfig }}",
+                    "kickstartImage": null,
+                    "bootImage": null,
+                    "staticDir": "cisco",
+                    "kickstartUri": "{{ api.server }}/{{ options.staticDir }}/{{ options.kickstartImage }}",
+                    "bootImageUri": "{{ api.server }}/{{ options.staticDir }}/{{ options.bootImage }}",
+                    "commands": [
+                        {
+                            "downloadUrl": "/api/1.1/templates/cisco-deploy-config-and-images.py"
```

Comment:
Second on this. In current practice, we use `/api/current` rather than `/api/2.0`.